cmd/kube-scheduler/app/options/deprecated.go

```diff
@@ -39,12 +40,29 @@ type DeprecatedOptions struct {
 	SchedulerName                  string
 }
 
+// TODO: remove these insecure flags in v1.24
+func addDummyInsecureFlags(fs *pflag.FlagSet) {
```

Comment:
Why do we need to keep them? they have been marked as deprecated since at least 1.17, we should be able to remove them. See example: https://github.com/kubernetes/kubernetes/pull/102239

Reply:
You are right. The flags removed. They were not wired to anything so it's just about removing `addDummyInsecureFlags`.

Reply:
Actually it seems like keeping those flags is the practice for this deprecation (both kube-apiserver and kcm have them)

Reply:
But I would still remove them. Just update the release notices to indicate that those two flags don't exist anymore